website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -102,14 +109,18 @@ export function MessageTableEntry({ message, enabled, highlight }: MessageTableE
         >
           {Object.entries(emojiState.emojis)
             .filter(([emoji]) => isKnownEmoji(emoji))
-            .map(([emoji, count]) => (
-              <MessageEmojiButton
-                key={emoji}
-                emoji={{ name: emoji, count }}
-                checked={emojiState.user_emojis.includes(emoji)}
-                onClick={() => react(emoji, !emojiState.user_emojis.includes(emoji))}
-              />
-            ))}
+            .sort(([emoji]) => -emoji)
+            .map(([emoji, count]) => {
+              console.log(emoji);
```

Comment:
could you please remove the console log

Reply:
Sorry I forgot it 😅, while I'm at it should I show the count only when it's greater than 0?

Reply:
kind of, yeah, but not for the `+1` or `-1`, otherwise we are back to square one

Reply:
![image](https://user-images.githubusercontent.com/113536090/216841747-a7672128-d7de-4f53-aa0b-2ee863ac9187.png)
done